easybuild/easyconfigs/o/OpenFOAM/OpenFOAM-2.3.1_external_paraview-4.3.1.patch

```diff
@@ -0,0 +1,52 @@
+#Use external Paraview installed in $EBROOTPARAVIEW directory
+# B. Hajgato Oct 28 2014
```

Comment:
@boegel: The patch was made that day... Sometimes it takes time to get from develop to master...
so what should be the date at the end?